install_config/install/prerequisites.adoc

```diff
@@ -362,6 +362,121 @@ SELINUX=enforcing
 SELINUXTYPE=targeted
 ----
 
+[[prereq-cloud-provider-considerations]]
+
+=== Cloud Provider Considerations
+
+*Set up the Security Group*
+
+When installing on AWS or OpenStack, ensure that you set up the security group.
+
+[cols="1,2"]
+|===
+|*All OpenShift Hosts*
+a|tcp/22 from host running the installer/Ansible
+
+|*etcd Security Group*
+a| - tcp/2379 from Masters
+- tcp/2380 from etcd hosts
+
+|*Master Security Group*
+a| - tcp/8443 from 0.0.0.0/0
+- tcp/53 from All OpenShift hosts
+- udp/53 from All OpenShift hosts
+
+|*Node Security Group*
+a| - tcp/10250 from Masters
+- tcp/4789 from Nodes
+
+|*Infrastructure Nodes*
+(ones that can host the `*openshift-router*`)
+a|- tcp/443 from 0.0.0.0/0
+- tcp/80 from 0.0.0.0/0
+
+|===
+
+If configuring ELBs for load balancing the Masters and/or routers, you also need
+to configure Ingress and Egress security groups for the ELBs appropriately.
+
+*Override Detected IP Addresses and Host Names*
+
+Some deployments will require that the user override the detected host names and
+IP addresses for the hosts. To see what the default values will be, run the
+`*openshift_facts*` playbook:
+
+====
+----
+ansible-playbook playbooks/byo/openshift_facts.yml
+----
+====
+
+Now, verify the detected common settings. If they are not what you expect them
+to be, you can override them.
+
+[cols="1,2"]
+|===
+|`*hostname*`
+a| - Should resolve to the internal IP from the instances themselves.
+- `*openshift_hostname*` will override.
+
+|`*ip*`
+a| - Should be the internal IP of the instance.
+- `*openshift_ip*` will override.
+
+|`*public_hostname*`
+a| - Should resolve to the external IP from hosts outside of the cloud.
+- Provider openshift_public_hostname will override.
+
+|`*public_ip*`
+a| - Should be the externally accessible IP associated with the instance.
+- `*openshift_public_ip*` will override.
+
+|`*use_openshift_sdn*`
+a| - Should be true unless the cloud is GCE.
+- `*openshift_use_openshift_sdn*` overrides.
```

Comment:
This is the only one in the series that says "overrides" instead of "will override".  I think there is something in the style guide that inveighs against "will", so the simple "overrides" of this line is actually preferable to hammering it into conformity w/ the others.